ui/src/app/kvm/components/KVMResourcesCard/KVMResourcesCard.tsx

```diff
@@ -14,12 +18,18 @@ import type { RootState } from "app/store/root/types";
 type Props = { id: Pod["id"] };
 
 const KVMResourcesCard = ({ id }: Props): JSX.Element => {
+  const dispatch = useDispatch();
   const pod = useSelector((state: RootState) =>
     podSelectors.getById(state, id)
   );
   const podVMs = useSelector((state: RootState) =>
     podSelectors.getVMs(state, id)
   );
+  const machinesLoading = useSelector(machineSelectors.loading);
+
+  useEffect(() => {
+    dispatch(machineActions.fetch());
```

Comment:
We should file a bug about being able to fetch just the machines for a KVM so that we don't have to load all the machines in the MAAS.

Reply:
Done https://bugs.launchpad.net/maas/+bug/1950108